client/client.go

```diff
@@ -200,7 +223,15 @@ func (c *Client) SetAccountSid(sid string) {
 	c.accountSid = sid
 }
 
-// Returns the Account SID.
+// AccountSid returns the Account SID.
 func (c *Client) AccountSid() string {
 	return c.accountSid
 }
+
+func (c *Client) SetOauth(oauth OAuth) {
+	c.oAuth = oauth
+}
+
+func (c *Client) OAuth() OAuth {
```

Comment:
I think we should just make this a property on the Client and remove the setters and getters. 

type Client struct {
    OAuth OAuth
}